data/lib/lib_copy_spider.lua

```diff
@@ -0,0 +1,65 @@
+function create_copy_spidertron(spidertron_name)
+    local constructron_name = "constructron-" .. spidertron_name
+    local constructron = table.deepcopy(data.raw["spider-vehicle"][spidertron_name])
+    constructron.name = constructron_name
+    constructron.collision_mask = {
+        "water-tile",
+        "colliding-with-tiles-only",
+        "not-colliding-with-itself"
+    }
```

Comment:
Why change the collision mask? For example the `ss-space-spidertron` has very specific collision masks.

Reply:
I probably copied this from the constructron code. Will be removed.